src/ray/util/logging.cc

```diff
@@ -120,8 +143,21 @@ void RayLog::StartRayLog(const std::string &app_name, RayLogLevel severity_thres
 #endif
 }
 
+void RayLog::UnInstallSignalAction() {
+  RAY_LOG(INFO) << "Uninstall signal handlers.";
+  static std::vector<int> installed_signals({SIGSEGV, SIGILL, SIGFPE, SIGABRT, SIGTERM});
```

Comment:
Where does this list come from? How do I check that it is complete?

Reply:
I think we should check the `RAY_USE_GLOG` macro here?